packages/api/tests/entities/Assignment.test.ts

```diff
@@ -19,13 +39,65 @@ describe('Assignment entity', () => {
     beforeEach(setup);
 
     it('should create an assignment successfully', async () => {
-      expect.assertions(2);
 
-      const assignmentInput = random.assignmentInput(moduleId);
-      const assignment = await TestClient.createAssignment(assignmentInput);
+      const assignment = await TestClient.createAssignment(assignment1);
 
       expect(assignment.id).toBeDefined();
-      expect(assignment).toMatchObject(assignmentInput);
+      expect(assignment).toMatchObject(assignment1);
+    });
+
+    it('should throw error if assignment already exist', async () => {
+
+        try {
+            const assignment = await TestClient.createAssignment(assignment1);
+        } catch(e) {
+            expect(e.message).toMatch(/unique constraint/i);
+        }
     });
   });
-});
+
+  describe('Query: getAssignments', () => {
+      beforeEach(setup);
+
+      it('should return the existing assignment', async () => {
+
+           await TestClient.createAssignment(assignment1);
+           const assignments = await TestClient.getAssignments(path.id);
+
+           expect(assignments).toBeArrayOfSize(1);
+           expect(assignments[0]).toMatchObject(assignment1);
+
+      });
+  });
+
+  describe('Mutation: updateAssignment', () => {
+      beforeEach(setup);
+
+      it('should update the existing assignment', async () => {
+
+          const startAssignment = await TestClient.createAssignment(assignment1);
```

Comment:
You could destructure the object here like:
const { id } = await ...
const update = {
  id,
  ...
}